liferay-workspace/modules/blade.indexerpostprocessor/src/main/java/com/liferay/blade/samples/indexerpostprocessor/BlogsIndexerPostProcessor.java

```diff
@@ -32,7 +32,10 @@
  */
 @Component(
 	immediate = true,
-	property = {"indexer.class.name=com.liferay.blogs.kernel.model.BlogsEntry"},
+	property = {
+		"indexer.class.name=com.liferay.blogs.kernel.model.BlogsEntry",
```

Comment:
By listing both class names, does this allow this postProcessor to work in both previous versions (before modulization) and after?